libc/include/llvm-libc-types/_Imaginary.h

```diff
@@ -0,0 +1,14 @@
+//===-- Definition of _Imaginary type -------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_TYPES__Imaginary_H
+#define LLVM_LIBC_TYPES__Imaginary_H
+
+typedef /* TODO */ _Imaginary;
```

Comment:
Does this emit a warning?

Reply:
It should. You see `_Imaginary` is a new keyword, introduced in the latest spec. I am discussing with @lntue on how to deal with it, will need to look into the latest versions of gcc and clang.

Reply:
Look like neither clang nor gcc support `_Imaginary` right now: https://godbolt.org/z/1Msrv3GsT
We can omit `_Imaginary` for now.

Reply:
Okay, I will comment the typedef out and add a TODO for it.

Reply:
I meant this file can be removed for now.  We shouldn't leave an empty header file.  You can create an issue to add `imaginary` macros when `_Imaginary` type is supported in clang or gcc.